src/types/demo.d.ts

```diff
@@ -16,3 +16,10 @@ export interface Demo {
     canIUseURL: string;
   };
 }
+
+// Adds EyeDropper to the global window object
+declare global {
+  export interface Window {
```

Comment:
Do you think export here is necessary?

Reply:
> Do you think export here is necessary?

I was getting this error "'Window' is defined but never used." from eslint and I thought instead of disabling the rule for this line, I might as well export it. I think some other people might need it. Or what do you think is the best approach?

Reply:
Oh, I see.
I just tried reproducing the same way and get the same 'error' by the linter. Seems like this is more like ESLint rules problem that this project repository set-up.

But to be honest, AFAIK there's no problem by the code itself. It just kinda weird for me at the first place when reading the code. Haha.